pkg/keys/ecdsa_test.go

```diff
@@ -0,0 +1,83 @@
+package keys
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"encoding/hex"
+	"encoding/json"
+	"io"
+	"strings"
+
+	fuzz "github.com/google/gofuzz"
+	"github.com/theupdateframework/go-tuf/data"
+	. "gopkg.in/check.v1"
+)
+
+type ECDSASuite struct{}
+
+var _ = Suite(&ECDSASuite{})
+
+func (Ed25519Suite) TestUnmarshalECDSA(c *C) {
+	priv, _ := ecdsa.GenerateKey(elliptic.P256(), strings.NewReader("00001-deterministic-buffer-for-key-generation"))
```

Comment:
Could we check errors in tests? There are a number of unchecked errors here.

Reply:
Done